### PR TITLE
[13.0][MIG] partner_external_map: Migration to 13.0

### PR DESCRIPTION
**Pull request by joao-p-marques**
Superseeds https://github.com/OCA/partner-contact/pull/813

- Applied pre-commit in separate [IMP] commit
- Applied suggestion from https://github.com/OCA/partner-contact/pull/813#discussion_r353771291
- Squashed administrative commits.

TT19784

ping @pedrobaeza @ernestotejeda

### CODE REVIEW

**Review by NL66278 — APPROVED** [review]
:+1: LGTM

File: partner_external_map/hooks.py [hunk 1]
```diff
@@ -11,7 +11,7 @@
 
 def set_default_map_settings(cr, registry):
     """Method called as post-install script
-    The default method on the field can't be used, because it would be executed
+    The default method on the field can"t be used, because it would be executed
```
Comment (pedrobaeza) [inline partner_external_map/hooks.py]: This change is not correct

Comment (joao-p-marques) [reply partner_external_map/hooks.py]: Changes made. Ty

**Pull request merged**